config/environments/development.rb

```diff
@@ -90,13 +89,13 @@
   config.action_mailer.perform_deliveries = true
   config.action_mailer.default_url_options = { host: config.app_domain }
   config.action_mailer.smtp_settings = {
-    address: "smtp.gmail.com",
-    port: "587",
-    enable_starttls_auto: true,
-    user_name: '<%= ENV["DEVELOPMENT_EMAIL_USERNAME"] %>',
-    password: '<%= ENV["DEVELOPMENT_EMAIL_PASSWORD"] %>',
-    authentication: :plain,
-    domain: config.app_domain
+    address: ENV["SMTP_ADDRESS"],
+    port: ENV["SMTP_ADDRESS"],
+    authentication: ENV["SMTP_AUTHENTICATION"],
+    user_name: ENV["SMTP_USER_NAME"],
+    password: ENV["SMTP_PASSWORD"],
+    domain: ENV["SMTP_DOMAIN"] || config.app_domain,
+    enable_starttls_auto: true
```

Comment:
Is this always mandatory or is it dependent on what `SMTP_AUTHENTICATION` contains?

Reply:
From the [doc](https://guides.rubyonrails.org/action_mailer_basics.html)
> :enable_starttls_auto - Detects if STARTTLS is enabled in your SMTP server and starts to use it. Defaults to true.

I don't think it depends on it? It will try TLS if able. We could even remove it too since it's redundant. I'm impartial though.

Reply:
Right, if it defaults to true we can just omit it from the conf, correct? Otherwise you'd need an additional env var...

Reply:
```suggestion
```
Let's remove it then!